tools/packaging/packaging.psm1

```diff
@@ -3049,7 +3049,7 @@ function New-MSIXPackage
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/ADOPTERS.md.https://shell.azure.com/ should work
```powershell
Failed to complete request to "https://shell.azure.com/". Response status code does not indicate success: 500 (Internal Server Error).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```